internal/rpc/proxy.go

```diff
@@ -339,8 +332,8 @@ func (p *clientProxy) start(ctx context.Context) error {
 		zapctx.Debug(ctx, "Reading on client connection")
 		msg := new(message)
 		if err := p.src.readJson(&msg); err != nil {
```

Comment:
reading at gorilla/websocket/json.go can return an error for two reasons:
- error on marshaling the json
- error on closing the connection.

I think your PR wants to hide error on closing connection, and that is fine from what I understand, but we are also masking errors on json marshaling, which could potentially lead to situation where the clients sends us non-json messages and we are not seeing the error.

Wouldn't be better to check here for marshaling error? And if not ignoring?

Reply:
Yes good point

Reply:
Fixed in #1345, I tried updating this PR but Github just hangs, I believe because this PR is still from my old private fork of JIMM.